packages/core/src/Menu/utils.ts

```diff
@@ -78,7 +78,7 @@ export function isPointInPolygon(point: Point, polygon: Polygon) {
   return inside
 }
 
-export function isPointerInGraceArea(event: PointerEvent, area?: Polygon) {
+export function isPointerInGraceArea(event: MouseEvent, area?: Polygon) {
```

Comment:
I would like to keep the event as `PointerEvent` 😁 as the function name suggest 'pointer'
same applies to the `isPointerInRect` function below.